index.js

```diff
@@ -160,4 +160,27 @@ const userLoggedIn = () => {
             }
         });
     });
+};
+
+// Lock routes to only one query param called status, have query param value be an allowed status, and allow an no query param
+const handleKitStatusReportsRoute = (auth, route) => {
+    const queryPart = route.split('?')[1];
+    const queryParams = new URLSearchParams(queryPart);
+
+    const requestedStatus = queryParams.get('status');
+    const status = requestedStatus?.trim()?.toLowerCase();
```

Comment:
It might be easier to save status in a variable or object (state manager?) that's accessible by functions.